### ISSUE TITLE
esc. key has to be pressed twice with --no-video

### ISSUE THREAD

**Issue by Arnaudv6** [mpv-player/mpv#5726]
when playing a file in the console (with --no-video)
the [esc.] key does not work the first time.
That's the old meta/raw-mode problem I guess.
Can this be amended?

**Comment by Arnaudv6** [comment]
Let me try again then: steps to reproduce:

1. under linux and Xorg, open a terminal (sakura in my case)
2. edit mpv/input.conf to bind an action to ESC in console for instance add this line:
`ESC quit`
3. play a file in console, for instance:
`mpv --no-video file.mp3`
4. presse [Esc.] key. This should work. It won't until you press it once more.

Can you reproduce this now?

**Comment by CounterPillow** [comment]
Can somewhat reproduce, but works for me if I hold Escape. Probably terminal key repeat weirdness.

Also, you still didn't post any version info, or a log.

**Comment by Arnaudv6** [comment]
I am using mpv 0.27.2, there is no log file.
In linux terminals, escape key is delayed so as to catch special keycodes AFAIK.

**Comment by CounterPillow** [comment]
Cool, so you're using an outdated version and didn't bother to read the issue template on how to provide a log file, and reported something that mpv can't really do anything about in the first place.

**Comment by Arnaudv6** [comment]
Terminals were not born in a day. Their default behaviour can be amended with settings and workarounds.
You're making quite some assumptions here @CounterPillow. Plus you're being impolite.
I give you a second chance at writing a constructive comment.
For instance you could show me the google search that points to your issue template.

**Comment by CounterPillow** [comment]
The issue template is the stuff you deleted from the issue description when you clicked on "New issue", you goof.

**Comment by Arnaudv6** [comment]
Deleted those lines five weeks ago... Searched for them on google. Should have opened another bug. But then you would complain about something else.
You reflect poorly on this comunity. I wish you can actually do something about this bug? Something else than bashing users. Then my efforts are not in vain.

**Comment by Arnaudv6** [comment]
> I for one...

So that's all that you brought with you @CounterPillow: your lack of respect for me.
You owe me respect. Bottom line.
You could also respect my efforts. As I wish to mention that I appreciate that you tried and reproduced the bug.
Since then, you had four chances, at trying to be constructive. I will happilly give you more.

**Comment by Arnaudv6** [comment]
Wow did we start on the wrong foot. I wish we could start again.

**Comment by Akemi** [comment]
i think that's it for now. as much as @CounterPillow can be a dislikable person he actually gave some quite good advice how to handle the situation, but instead of listening the general behaviour is discussed and not the issue at hand? in the first place your owe us developers a bit of respect by not blindly deleting the issue template and responsibly filling it out.

so as you wish. feel free to open a new issue with a clean slate.
